install/install.qmd

```diff
@@ -105,96 +106,192 @@ For more information, see the
+then follow instructions in the README file.
 :::::
 
 ::::
 :::
 
 <script src="installer.js"></script>
 
-## In-Depth Instructions
 
-### Platform
 
-#### Linux
+::: {.callout-note}
+Conda simplifies the installation process by ensuring that all required libraries and tools are compatible with each other
+and is available for Linux, Mac, and Windows platforms.
+
+You can either install [miniconda](https://docs.conda.io/en/latest/miniconda.html), a free, minimal installer for conda
+or you can get the full [Anaconda](https://docs.continuum.io/anaconda/) system
+which provides graphical installer wizards for [MacOS](https://www.anaconda.com/products/distribution#macos){target="_blank"}
+and [Windows](https://www.anaconda.com/products/distribution#windows){target="_blank"} users.
+
+Julia users can install [Conda.jl](https://github.com/JuliaPy/Conda.jl){target="_blank"}.
+:::
+
+## Prerequisite:  C++17 toolchain
+
+
+Stan requires a C++17 compiler and the GNU Make utility.
+
+* On Linux, these are bundled into the meta-package `build-essential`.
+To install, run command:\
+`sudo apt-get install build-essential`
+
+
+* On Mac, the Clang compiler and GNU Make are included with Xcode, the Apple toolset for software developers.
+Install Xcode from the App Store and then run command:\
+`xcode-select --install`
+
+* On Windows 10, there are two ways to get a Stan-compatible C++ toolchain:
+   + Use the conda installer for CmdStan, CmdStanPy or CmdStanR or RStan, since these packages all include the required toolchain.
+   + R Users can get [Rtools](https://cran.r-project.org/bin/windows/Rtools/){target="_blank"} from CRAN; the CmdStanR `install_cmdstan()` function will do this automatically.
+
+
+## Local CmdStan installations for CmdStanPy, CmdStanR, and Stan.jl
+
+CmdStanPy, CmdStanR, and Stan.jl and require a local CmdStan installation.
+Both CmdStanPy and CmdStanR provide method `install_cmdstan` to do this from within Python or R;
+and CmdStanPy also provides this as a command-line function.
+See the online documentation:
+
+* CmdStanPy:  [install_cmdstan function](https://mc-stan.org/cmdstanpy/installation.html#function-install-cmdstan)
+* CmdStanR: [install_cmdstan function](https://mc-stan.org/cmdstanr/reference/install_cmdstan.html)
+* Stan.jl [build instructions](https://github.com/StanJulia/Stan.jl/blob/master/INSTALLING_CMDSTAN.md)
+
+The default installation location is in the user's home directory and is named `.cmdstan` (a hidden directory).
+This directory contains one or more versions of CmdStan.
+
+Both CmdStanPy and CmdStanR provide the following functions:
+
+* `rebuild_cmdstan` - rebuild the specified release.  On Mac, often required after an Xcode update.
+* `show_versions` - shows all installed CmdStan versions
+* `cmdstan_path` - shows which version of CmdStan is being used
+* `set_cmdstan_path` - specify which version of CmdStan to use.
+
+## Other Stan Interfaces {#other-packages}
```

Comment:
Add `brms`, `rstanarm`, `rethinking` as higher-level interfaces (or perhaps a new section for these?)

Reply:
added.